src/components/v1/V1Project/modals/TransferUnclaimedModal.tsx

```diff
@@ -0,0 +1,113 @@
+import { t, Trans } from '@lingui/macro'
+import { Form, Modal } from 'antd'
+import { useWatch } from 'antd/lib/form/Form'
+import InputAccessoryButton from 'components/InputAccessoryButton'
+import { EthAddressInput } from 'components/inputs/EthAddressInput'
+import FormattedNumberInput from 'components/inputs/FormattedNumberInput'
+import { V1ProjectContext } from 'contexts/v1/projectContext'
+import { isAddress } from 'ethers/lib/utils'
+import useUnclaimedBalanceOfUser from 'hooks/v1/contractReader/UnclaimedBalanceOfUser'
+import { useTransferTokensTx } from 'hooks/v1/transactor/TransferTokensTx'
+
+import { FC, useContext, useState } from 'react'
+import { formatWad, fromWad, parseWad } from 'utils/format/formatNumber'
+import { tokenSymbolText } from 'utils/tokenSymbolText'
+
+interface TransferUnclaimedModalProps {
+  visible?: boolean
+  onCancel?: VoidFunction
+  onConfirmed?: VoidFunction
+}
+
+export const TransferUnclaimedModal: FC<TransferUnclaimedModalProps> = ({
+  visible,
+  onCancel,
+  onConfirmed,
+}) => {
+  const unclaimedBalance = useUnclaimedBalanceOfUser()
+  const { tokenSymbol } = useContext(V1ProjectContext)
+  const [loading, setLoading] = useState<boolean>()
+  const [form] = Form.useForm<{ amount: string; to: string }>()
+  const transferUnclaimedTokensTx = useTransferTokensTx()
+
+  const amount = useWatch('amount', form)
+  const address = useWatch('to', form)
+
+  const transferTokens = async () => {
+    setLoading(true)
+
+    transferUnclaimedTokensTx(
+      {
+        to: address,
+        amount: parseWad(amount),
+      },
+      {
+        onConfirmed: () => {
+          form.resetFields()
+          setLoading(false)
+          onConfirmed?.()
+        },
+        onError: () => {
+          setLoading(false)
+        },
+      },
+    )
+  }
+
+  const tokenTextShort = tokenSymbolText({
+    tokenSymbol,
+    plural: true,
+  })
+
+  return (
+    <Modal
+      title={t`Transfer unclaimed ${tokenTextShort}`}
+      visible={visible}
+      confirmLoading={loading}
+      onOk={() => {
+        transferTokens()
+      }}
+      onCancel={() => {
+        form.setFieldsValue({
+          amount: undefined,
+          to: undefined,
+        })
+        onCancel?.()
+      }}
+      okText={t`Transfer ${tokenTextShort}`}
+      okButtonProps={{
+        disabled: parseWad(amount).eq(0) || !isAddress(address),
```

Comment:
This is a decent approach, but I think, mainly for the sake of consistency with other forms throughout the app, it might be even better to go with this approach:
- pass validator functions as props into the two `<Form.Item`s. (e.g. see `validateTotalDistributions`)
- put an `await form.validateFields()` at the start of the `transferTokens` function
- Never disable the button 

The advantage of this approach is that we can show users why the form is not valid if they try to submit. It's not a big deal in this case since the form is so small, but I think it's a good practice to get into the habit of for future forms you will build. 

Let me know if this doesn't make sense, or if you have any trouble implementing.

Reply:
It is definitely healthier to allow users to click and if items are not filled show the message, thanks for this!

I've applied this behavior in the latest commit, but there is still inconsistency throughout the app

- Redeem and Burn modal disables the button
- Claim and Transfer unclaimed tokens enable button but display a message to a user.

I can create an issue after this PR and make all these modals consistent, wdyt?

Reply:
For sure, that would be great.